LICENSE

```diff
@@ -1,6 +1,44 @@
+FlowLauncher are licensed under the terms of the MIT license, reproduced below.
+
+
+A. HISTORY OF THE SOFTWARE
+==========================
+
+- Copyright (c) 2015-2019 Wox. All rights reserved.
+- Copyright (c) 2019-2020+ Flow-Launcher. All rights reserved.
+
+
+B. FlowLauncher LICENSE
+==========================
+
+The MIT License (MIT)
+
+Copyright (c) 2019-2020+ Flow-Launcher. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of
+this software and associated documentation files (the "Software"), to deal in
+the Software without restriction, including without limitation the rights to
+use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+the Software, and to permit persons to whom the Software is furnished to do so,
+subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+
+C. Wox LICENSE
+==========================
+
 The MIT License (MIT)
 
-Copyright (c) 2015 Wox
+Copyright (c) 2020 Wox https://github.com/Wox-launcher/Wox
```

Comment:
why is the year 2020 here but 2015-2019 above? Also I'm not sure modifying the original license is a good idea.

Reply:
For this, I'm also a little confused.

Here is the license changing history of the Wox https://github.com/Wox-launcher/Wox/commits/master/LICENSE

And the last change is **2020**.

https://github.com/Wox-launcher/Wox/blob/58ae4642e087c62dd129f40652d1cac117744779/LICENSE#L3

For this condition, which should we choose?